packages/quantic/cypress/integration/case-classification/case-classification.cypress.ts

```diff
@@ -0,0 +1,385 @@
+import {configure} from '../../page-objects/configurator';
+import {
+  interceptSearch,
+  mockCaseClassification,
+  mockSfPicklistValues,
+  interceptClassificationsIndefinitely,
+} from '../../page-objects/search';
+import {CaseClassificationExpectations as Expect} from './case-classification-expectations';
+import {CaseClassificationActions as Actions} from './case-classification-actions';
+import {scope} from '../../reporters/detailed-collector';
+import {fetchClassifications} from '../../page-objects/actions/action-fetch-classifications';
+
+interface CaseClassificationOptions {
+  maxChoices: number;
+  label: string;
+  required: boolean;
+  selectPlaceholder: string;
+  messageWhenValueMissing: string;
+}
+
+describe('quantic-case-classification', () => {
```

Comment:
It might be a stupid test to do, but does it make sense to test that the suggestions appear in the order that they were received from the API?

I only see tests that checks the correct number of suggestions are displayed for example. Would it make sense to also test in the order in which they get displayed?

Reply:
Absolutely ! I will add such tests 👌